tests/core/block/test_prefix_caching_block.py

```diff
@@ -315,6 +315,59 @@ def test_get_num_free_blocks(num_blocks: int, block_size: int, seed: int):
                                                        i)
             allocator.free(block)
 
+    @staticmethod
+    @pytest.mark.parametrize("num_blocks", [4])
+    @pytest.mark.parametrize("block_size", [8])
+    def test_get_num_blocks_touched(num_blocks, block_size):
```

Comment:
Can we merge two test_get_num_blocks_touched functions and control them by `enable_caching` parameter? Or at least make their name different?

Reply:
Done, made the test name different.